README.md

```diff
@@ -27,7 +27,7 @@ The project tree is set up as follows:
  </tr>
  <tr>
   <td>web/</td>
-  <td>java and resource files that are used in the webapp/war file.</td>
+  <td>java and resource files that are used in the web-app/war file.</td>
```

Comment:
@SarthakA Please do not change this at it is wrong. That is the path to the `.war` file so you can't change this